includes/block-patterns/course/video-hero.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * Video Hero pattern.
+ *
+ * @package sensei-lms
+ */
+
+return [
+	'title'      => __( 'Video Hero', 'sensei-lms' ),
+	'categories' => [ \Sensei_Editor_Wizard::PATTERNS_CATEGORY ],
+	'blockTypes' => [ \Sensei_Editor_Wizard::POST_CONTENT_BLOCK_TYPE ],
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var(\u002d\u002dwp\u002d\u002dcustom\u002d\u002dspacing\u002d\u002dlarge, 8rem)","bottom":"var(\u002d\u002dwp\u002d\u002dcustom\u002d\u002dspacing\u002d\u002dlarge, 8rem)"}},"elements":{"link":{"color":{"text":"var:preset|color|secondary"}}}},"backgroundColor":"foreground","textColor":"secondary"} -->
```

Comment:
Just a thought, but I wonder if it makes sense for these to be PHP template files rather than strings? So it would look something like:

```php
<!-- wp:group ... -->
<h1 ...><?php esc_html_e( 'Welcome ...', 'sensei-lms' ) ?></h1>
...
<!-- /wp:group -->
```

Not sure how much benefit there is, but it seems like it might be a bit cleaner than having it in a string? 🤷‍♂️

Reply:
Should we consider adding translations also to the texts inside the patterns?

Reply:
Interesting. Just seems weird for it to be a great big string. But I don't have any serious objection to that 🤷‍♂️

Reply:
Inpired by @aaronfc [PR](1247-gh-Automattic/sensei-pro), I updated it here, following your suggestion: https://github.com/Automattic/sensei/pull/5206/commits/cc6c0bc7ff304c9675bcbccf73828c76f6429459

I was probably overthinking this, and just a `require` was enough to extract the file.